modules/ocf_desktop/files/xsession/print-notify-handler

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+import functools
+import os
+from configparser import ConfigParser
+
+import redis
+
+
+redis_connection = functools.partial(redis.StrictRedis,
+                                     host='broker',
+                                     port=6378,
+                                     ssl=True,
+                                     db=0,
+                                     password=None)
+
+
+def subscribe(host, password, *channels):
+    rc = redis_connection(host=host, password=password)
+    sub = rc.pubsub(ignore_subscribe_messages=True)
+    sub.subscribe(channel for channel in channels)
+    return sub
+
+
+BROKER_AUTH = '/opt/share/broker/broker.conf'
```

Comment:
This should go further up in the file, definitely above the `subscribe` method anyway.